Cargo.toml

```diff
@@ -10,3 +10,6 @@ members = [
 
 [profile.dev]
 opt-level = 2
+
+[patch.crates-io]
+rand_core = { git = "https://github.com/rust-random/rand_core.git" }
```

Comment:
Do you really want to use a repo without specifying revision or branch? It could break frequently (we have more changes planned). I usually specify `rev = "$REVISION"`.

Reply:
We commit Cargo.lock, so without `cargo update` the revision will stay pinned.